CMakeLists.txt

```diff
@@ -1,8 +1,6 @@
 cmake_minimum_required(VERSION 3.1.0 FATAL_ERROR)
 project(MAPBOX_BASE LANGUAGES CXX C)
 
-enable_testing()
```

Comment:
Why removing this?

Reply:
It is moved into test directory: https://github.com/mapbox/mapbox-base/blob/kdn-dev/test/CMakeLists.txt#L1

Reply:
No need for this, `enable_testing()` can be overriden with `BUILD_TESTING` option (see https://cmake.org/cmake/help/latest/command/enable_testing.html for details).

Reply:
I've moved it into root `CMakeLists.txt`. Why we need to replace it with `BUILD_TESTING`? I have read documentation and didn't understood.

Reply:
It means you don't need to add an extra option e.g. if you want to disable testing, you can issue `cmake -DBUILD_TESTING=0`